webapp/src/app.tsx

```diff
@@ -1760,6 +1773,17 @@ export class ProjectView
             'full-abs'
         ];
         const rootClasses = sui.cx(rootClassList);
+
+        if (this.state.hasError) {
+            return <div id="root" className="ui middle aligned center aligned grid" style={{ height: '100%', alignItems: 'center' }}>
+                <div className="ui raised segment inverted purple">
+                    <h2>{lf("Oops") + "  😞😞😞"}</h2>
+                    {lf("This doesn't happen often, but looks like we've run into a problem..")}
```

Comment:
Let's keep the wording simpler. Something like "Sorry we detected a problem and we will reload the editor in a few seconds." Or something better but let's not make claims about frequency.

Reply:
Haha fair enough, I was trying to be cute